plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.scss

```diff
@@ -27,6 +27,21 @@
 		&.is-active {
 			border-color: var(--wp-admin-theme-color);
 		}
+		.awaiting-mod {
```

Comment:
Can we make the class name more meaningful? At first glance, it is not clear what it is.

Reply:
Also, notice the gap between the tab's name and the badge in the original design.

![image](https://github.com/woocommerce/woocommerce/assets/12912469/b02fcb53-a207-4ce0-980c-3a623cab4ca4)

Reply:
> Can we make the class name more meaningful? At first glance, it is not clear what it is.

Can we use this one `woocommerce-badge-color-red` ?

Reply:
> Also, notice the gap between the tab's name and the badge in the original design.

Addressed the gap size [e84baea](https://github.com/woocommerce/woocommerce/pull/46034/commits/e84baea6b63fbecce8b0cad0734fd1e556a7554a)

I noticed that WC core has its own badge style, which can be seen on the Home sub-menu under the WooCommerce menu. That's why I didn't alter the width and border-radius to ensure compatibility with the core design. What do you think?